common/config/config.json.template

```diff
@@ -11,16 +11,16 @@
   "liveDvrWindow" : 100,
   "logLevel" : "debug",
   "logToConsole" : true,
-  "logFileName" : "/var/log/dvr_worker.log",
+  "logFileName" : "/var/log/kDvrWorker.log",
```

Comment:
Please follow our path standards:
/opt/kaltura/log/dvr/worker.log and /opt/kaltura/log/dvr/server.log

+@Ronnysh